apps/CMakeLists.txt

```diff
@@ -241,6 +241,12 @@ if(build)
   PCL_ADD_INCLUDES("${SUBSYS_NAME}" "${SUBSYS_NAME}" ${incs})
   PCL_ADD_INCLUDES("${SUBSYS_NAME}" "${SUBSYS_NAME}/impl" ${impl_incs})
 
-  add_custom_target(${SUBSYS_NAME} DEPENDS ${PCL_APPS_ALL_TARGETS})
+  if(CMAKE_GENERATOR_IS_IDE)
+      set(SUBSYS_TARGET_NAME ALL_APPS)
+  else()
+      set(SUBSYS_TARGET_NAME ${SUBSYS_NAME})
+  endif()
+  add_custom_target(${SUBSYS_TARGET_NAME} DEPENDS ${PCL_APPS_ALL_TARGETS})
+  set_target_properties(${SUBSYS_TARGET_NAME} PROPERTIES FOLDER "Apps")
```

Comment:
Here the `FOLDER` property is set unconditionally, however on next occasions only if `USE_PROJECT_FOLDERS` is on. If you can confirm that setting `FOLDER` while the global property `USE_FOLDERS` is off does not actually create folders, then let's set this property always unconditionally to make code less convoluted.

Reply:
I forgot to add the check. I'm also considering we should actually set  `USE_PROJECTS_FOLDERS` on by default because things looks like a mess on  IDEs without it. See https://github.com/PointCloudLibrary/pcl/pull/2673#issuecomment-443512458

Reply:
Agree about the default. But as I said, maybe without check will also work.